README.md

```diff
@@ -144,10 +144,11 @@ Then bring down the project code:
 1. Run `azd init -t azure-search-openai-demo`
     * note that this command will initialize a git repository and you do not need to clone this repository
 
-### Deploying from scratch
+### Deploying
 
-Execute the following command, if you don't have any pre-existing Azure services and want to start from a fresh deployment.
+Follow these steps to provision Azure resources and deploy the application code:
 
+1. (Optional) This is the point where you can customize the deployment by setting environment variables, in order to [use existing resources](#using-existing-azure-resources), [enable optional features](#enabling-optional-features), or [deploy to free tiers](docs/deploy_lowcost.md).
```

Comment:
My thought here is to call out that any customization should happen first. My other thought is that we could move "using existing" and "enabling optional features" into its own doc, to significantly shorten this README if we wanted. But that's a larger change, so I just made this one.

Reply:
I'm sold, let's do it